generators/chipyard/src/main/scala/config/BoomConfigs.scala

```diff
@@ -96,29 +92,7 @@ class DualSmallBoomConfig extends Config(
   new freechips.rocketchip.subsystem.WithNoSlavePort ++
   new freechips.rocketchip.subsystem.WithInclusiveCache ++
   new freechips.rocketchip.subsystem.WithNExtTopInterrupts(0) ++
-  new boom.common.WithSmallBooms ++
-  new boom.common.WithNBoomCores(2) ++                         // 2 boom cores
-  new freechips.rocketchip.subsystem.WithCoherentBusTopology ++
-  new freechips.rocketchip.system.BaseConfig)
-
-class SmallRV32BoomConfig extends Config(
```

Comment:
Why delete this config?

Reply:
Supporting Rv32 in boom is too annoying.